src/common/wal/Cargo.toml

```diff
@@ -18,6 +18,8 @@ common-telemetry.workspace = true
 futures-util.workspace = true
 humantime-serde.workspace = true
 rskafka.workspace = true
+rustls = { version = "0.23", default-features = false, features = ["ring", "logging", "std", "tls12"] }
+rustls-pemfile = "2.1"
```

Comment:
Should we update the these deps in the `servers` crate? @sunng87

Reply:
I remember it was tonic blocking us from the upgrade. We need to update tonic to 0.12.

Reply:
I think we can upgrade rustls of servers in another PR. It requires upgrade of proto module as well and could take some time.